core/kazoo_auth/priv/couchdb/providers/google.json

```diff
@@ -1,9 +1,30 @@
 {
     "_id": "google",
-    "enabled": true,
```

Comment:
Why removing this? I added this key here to allow disable or enable a specific provider.